src/styles/abstracts/mixin.tsx

```diff
@@ -0,0 +1,45 @@
+export const fontMontserratRegular = () => {
+  return `
+    font-family: "Montserrat", Helvetica, Arial, sans-serif;
+    font-weight: 400;
+  `;
+};
+
+export const fontMontserratMedium = () => {
+  return `
+    font-family: "Montserrat", Helvetica, Arial, sans-serif;
+    font-weight: 500;
+  `;
+};
+
+export const fontMontserratSemibold = () => {
+  return `
+    font-family: "Montserrat", Helvetica, Arial, sans-serif;
+    font-weight: 600;
+  `;
+};
+
+export const fontSourcesansproRegular = () => {
+  return `
+    font-family: "Source Sans Pro", Helvetica, Arial, sans-serif;
+    font-weight: 400;
+  `;
+};
+
+export const fontSourcesansproBold = () => {
+  return `
+    font-family: "Source Sans Pro", Helvetica, Arial, sans-serif;
+    font-weight: 700;
+  `;
+};
+
+const pxRem = (size: number, base = 16) => {
+  return (size / base) * 1 + "rem";
+};
+
+export const fontSize = (size = 16) => {
```

Comment:
what's the use of this ?

Reply:
is to have a fallback to px value, if rem is somehow not supported in IE. it will look like these:

supported:
~~font-size: 16px;~~
font-size: 1rem;

not supported:
font-size: 16px;
~~font-size: 1rem;~~

Reply:
ah ok, please add a comment :)